solc/CommandLineInterface.cpp

```diff
@@ -728,7 +728,14 @@ bool CommandLineInterface::compile()
 			m_compiler->setParserErrorRecovery(m_options.input.errorRecovery);
 		}
 
-		bool successful = m_compiler->compile(m_options.output.stopAfter);
+		bool successful = m_compiler->compile(m_options.output.stopAfter, false);
+		if (successful)
+		{
+			std::set<std::string> usedLibs = m_compiler->link();
+			for (auto lib: m_options.linker.libraries)
```

Comment:
You should use `const auto&` here since a plain `auto` will copy the underlying string value.

Reply:
Got it!